libraries/testing/tester.cpp

```diff
@@ -1253,7 +1253,7 @@ namespace eosio::testing {
                ("pop", pop.to_string()));
       }
 
-      control->set_node_finalizer_keys(local_finalizer_keys);
```

Comment:
Why is the `enable_immediate_voting` needed?

Reply:
I believe because the blocks are produced with an old (near genesis) timestamp.

Reply:
That's odd! Shouldn't we fix that instead of having this `enable_immediate_voting` workaround?

Reply:
We could look into it, but I suspect it would require a newly generated genesis for each test.

Reply:
Oh, I get why this happens. We want blocks to be produced every 500ms by default, so that each producer does its 12 blocks, regard;less of the clock time.